.travis.yml

```diff
@@ -31,6 +31,7 @@ install:
   - eval "$(gimme)"
   - curl https://raw.githubusercontent.com/golang/dep/7d5cd199ce454707f81c63b7ea4299151b8b981d/install.sh | sh
   - npm install -g npm@6.4.0 yarn@1.9.4
+  - npm install -g bower@1.8.2
```

Comment:
Simply add this to the previous line (as a first item, sorted alphabetically), so `npm install` is only called once.